README.md

```diff
@@ -22,6 +22,42 @@ _Manage > Overview > Add an object to this Collection > Give it a Name > Ingest_
 
 The base ZIP/directory preprocessor can be called as a drush script (see `drush help islandora_newspaper_batch_preprocess` for additional parameters):
 
+```
+drush help islandora_batch_scan_preprocess
+
+Examples:
+drush -v --user=admin --uri=http://digital.library.yorku.ca islandora_batch_scan_preprocess --content_models=islandora:sp_large_image_cmodel --parent=yul:F0433 --parent_relationship_pred=isMemberOfCollection --type=directory --target=/tmp/batch_ingest
+
+
+Options:
+
+--content_models 
+Supports one or multiple comma-separated content models which are all applied to each ingested object.
+
+--namespace
+Namespace of objects to create. Defaults to namespace specified in Fedora configuration.
+
+--parent
+The collection to which the generated items should be added. Defaults to the root Islandora repository PID.
+
+--parent_relationship_pred 
+The predicate of the relationship to the parent. Defaults to "isMemberOfCollection".
+
+--parent_relationship_uri 
+The namespace URI of the relationship to the parent. Defaults to "info:fedora/fedora-system:def/relations-ex ternal#".
```

Comment:
Typo, the is an errant space in `relations-ex ternal#`

Reply:
thanks! I'll fix that.